AndelaOSP/andela-societies-frontend
# Ft allow fellows comment on logged activity#159101182

## Resolves [#159101182](https://www.pivotaltracker.com/story/show/159101182)

## Description
As a logged in user,
- WHEN I am on the My activities page.
- AND there is a logged activity that requires additional details.
- THEN I should be able to click on the activity card to add more information.

## How to test
Once logged in as a user (Fellow)
- Navigate to My Activities page.
- Click on any card that has the status of (in review or pending).
- Ensure the cards with the statuses above are clickable.
- Ensure a modal pops up containing the activity information
- Fill in the required information on the activity card.
- Click on the update button. Ensure messages are correctly seen and the modal closes.

## Screenshots

<img width="1075" alt="screen shot 2018-07-27 at 4 28 25 pm" src="https://user-images.githubusercontent.com/26232631/43330113-2d886d9c-91ba-11e8-8d7f-73c4a10347ef.png">
